<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labe_Parity">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labe_Data_Bits">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data Bits:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Serial_Port">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Baud_Rate">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Baud Rate:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Stop_Bits">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Data_Bits">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Serial_Port">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Serial Port: </string>
    </property>
   </widget>
   <widget class="QLabel" name="labe_Stop_Bitsl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop Bits:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Open_Close_Serial">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open Serial Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Parity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Baud_Rate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Start">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Stop">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
